CorePayments/src/main/java/com/paypal/android/corepayments/RestClient.kt

```diff
@@ -1,23 +1,27 @@
 package com.paypal.android.corepayments
 
-import com.paypal.android.corepayments.analytics.AnalyticsEventData
 import java.net.URL
 import java.util.Locale
 
-internal class HttpRequestFactory(private val language: String = Locale.getDefault().language) {
+class RestClient internal constructor(
+    private val configuration: CoreConfig,
+    private val http: Http = Http(),
+    private val language: String = Locale.getDefault().language
```

Comment:
Tangent - we're always setting this to `en_US` on iOS too. Worth looking into if that will restrict usage in other countries at all.

Reply:
Yeah i'm not sure what affect Locale has on PayPal's web services, but thinking about it now I wouldn't be surprised if we're required to allow the merchant to specify in the future.

Reply:
Yeah, we allow merchants to specify via `localeCode` on BT iOS and Android. Harder to test other locales, but we can switch this value to another locale and see if it always defaults to that even if device is US based. That will at least show us if users in other countries would always get the US based locale.

Reply:
Made a ticket: DTNOR-972

Reply:
Good question - on BT we can pass it into PayPal/MXO (though I don't actually know if MXO respects it). I assume we want to use it for everything? But actually don't know from a product perspective what the goal is, so may be good to clarify with product in the follow up ticket!